FV3/conf/configure.fv3.gnu_docker_serialize

```diff
@@ -2,8 +2,8 @@
 ############
 # commands #
 ############
-FC = FC=gfortran mpifort
```

Comment:
I will need to make sure I can still serialize the data with these changes, but there's not reason to think this would have an impact?

Reply:
Did you want additional testing to check this?

Reply:
That's ok, we can sort through any issues when we make the next data experiments, which will be very soon, next week.